test/compile/format.test.ts

```diff
@@ -82,6 +82,10 @@ describe('Format', () => {
       expect(numberFormat(QUANTITATIVE, undefined, {numberFormat: 'd'})).toBe('d');
     });
 
+    it('should use normalized number format for quantitative scale with stack: "normalize"', () => {
+      expect(numberFormat(QUANTITATIVE, undefined, {numberFormat: 'd', normalizedNumberFormat: 'c'}, true)).toBe('c');
```

Comment:
For test's readability,  should we change `numberFormat` to use object params instead?